common/components/GasSlider/components/AdvancedGas.tsx

```diff
@@ -52,7 +52,7 @@ class AdvancedGas extends React.Component<Props> {
           />
         </div>
 
-        <div className="col-md-4 col-sm-6 col-xs-12 AdvancedGas-gasLimit">
+        <div className="col-md-4 col-sm-6 col-xs-12 AdvancedGas-boostZIndex AdvancedGas-gasLimit">
```

Comment:
I actually realized there's a much simpler fix to this, which is to add `col-xs-12` to the columns that have `col-sm-12` and `col-md-12`. I guess Bootstrap wants you to specify the smallest column size. I'd prefer this over messing with z-index, that always comes back to bite you.

Reply:
Great suggestion, that's definitely the correct approach! Just pushed the change.